src/store/stub/store.py

```diff
@@ -42,4 +49,29 @@ def get_publishers(self) -> dict:
         return self.publishers
     
     def get_topics(self) -> dict:
-        return self.topics
+        return self.themes
+
+
+class StubMetadataStore(BaseCsvStore):
+    """
+    A stub of a store that returns representative csv from
+    files stored on disk.
+    """
+    def setup(self):
+        self.datasets = {}
+
+        content_dir = Path(Path(__file__).parent / "csv/cipd/2022-3")
+
+        with open(Path(content_dir / "dataset.csv").absolute()) as f:
+            self.datasets["dataset"] = csv.reader(f)
```

Comment:
so the field ("dataset") in this case needs to be matched by what you're passing in on line 75, so this should be 
```
self.dataset["cipd/2022-3/dataset"] = csv.reader(f)
```

do also not the comment on naming of therse csv files as well please.